src/main.rs

```diff
@@ -344,16 +349,39 @@ pub enum RecordsOut {
 }
 
 impl RecordsIn {
-    pub async fn into_reader(&self) -> std::io::Result<Box<dyn AsyncBufRead + Send + Unpin>> {
+    pub async fn into_lines_stream(
```

Comment:
`into_reader` was ok as a name here (consistent with `RecordsOut`)

Reply:
Doesn't sound right since it isn't a reader anymore. Maybe return lines stream from `RecordsOut` as well and rename it?

Reply:
yup, sg

Reply:
oh wait, let's not do any of it. `Stdout` is a writer.